Sources/ParseSwift/Types/ParseError.swift

```diff
@@ -352,6 +352,11 @@ public struct ParseError: ParseType, Decodable, Swift.Error {
          */
         case xDomainRequest = 602
 
+        /**
+         Error code indicating the Parse Swift struct is invalid.
+         */
+        case invalidStruct = 603
```

Comment:
I don't see `603` as a code in the JS SDK or server. What happens if another error code is put in it's place at the server and JS SDK level? It seems this number should be something stable or internal to the SDK. Unless you plan to introduce this error at the JS and server level?

Reply:
Changing to `case invalidStruct` doesn't resolve the problem. If you are not adding this error to server and JS SDK, then the liens added to ParseError should be removed and you should be using `.unknownError` as it's currently the way internal SDKs errors are handled. You current solution can cause a major issue when a new error is introduced at the server level as it will decode improperly when using the Swift SDK

Reply:
Ah right, I understand!